docs/reference/configuration/keywords.rst

```diff
@@ -908,6 +908,20 @@ and/or history matching project.
         The various prior distributions available for the ``GEN_KW``
         keyword are described :ref:`here <prior_distributions>`.
 
+        It is also possible to skip the templating functionality of ``GEN_KW``, it also allows
+        for two positional arguments;
+
+        ::
+
+                GEN_KW  ID  priors.txt
+
+        where :code:`ID` is an arbitrary unique identifier,
+        and :code:`priors.txt` is a file containing a list of parametrized keywords
+        and a prior distribution for each.
+
+        this way ``GEN_KW`` will not create an output file for each realization, but instead
+        add the parameter values to a file located in runpath called: ``parameters.json``
```

Comment:
users are more familiar with parameters.txt, which I guess it is also outputted to.

Reply:
Yes, it was intentional to omit it though, because we did at one point say it was deprecated. But I guess that ship has sailed and we should just add it.